libindy/src/api/anoncreds.rs

```diff
@@ -83,46 +85,54 @@ pub extern fn indy_issuer_create_and_store_claim_def(command_handle: i32,
 #[no_mangle]
 pub extern fn indy_issuer_create_and_store_revoc_reg(command_handle: i32,
                                                      wallet_handle: i32,
+                                                     tails_writer_config_handle: i32,
```

Comment:
I can't find public API to find this handle. Could you point me?

Reply:
I have already changed it.
Now it look like: 
pub extern fn indy_issuer_create_and_store_revoc_reg(command_handle: i32,
                                                                                         wallet_handle: i32,
                                                                                         tails_writer_type: *const c_char,
                                                                                         tails_writer_config: *const c_char,
                                                                                         issuer_did: *const c_char,
                                                                                         schema_json: *const c_char,
                                                                                         max_claim_num: u32,
                                                                                         issuance_by_default: bool,